### PR TITLE
Task3 layout

### PR DESCRIPTION
**Pull request by denisbaranchik**
layout created

### CODE REVIEW

**Comment by MaksimViarenich** [comment]
You did a good job! I understand that it's not easy to write a good html+css.
We can discuss on the lesson the most common mistakes and if you have any questions you are welcome. You. have too much CSS, it can be decreased twice

**Comment by MaksimViarenich** [comment]
You can set in IntelliJ Code style for your HTML/CSS and the fix code style by `Ctrl+Shift+L`
Снимок экрана 2020-03-02 в 22 33 58

**Comment by denisbaranchik** [comment]
Thanks! Let's discuss everything at the lesson first

**Comment by denisbaranchik** [comment]
It seems i fixed everything you've mentioned:

-Get rid of redundant divs in html
-Get rid of style in html
-Made better checkboxes
-Made buttons with <button>
-Fixed divs overflow
-Updated css length and style

File: UI/index.html [hunk 1]
```diff
@@ -0,0 +1,186 @@
+<!DOCTYPE html>
+<html>
+
```
Comment (MaksimViarenich) [inline UI/index.html]: Minor. Remove one empty line

File: UI/index.html [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+            <div class="filters-title text-style-1">
+                Filters
+            </div>
+
+            <div class="container filters-container">
+
+                <div class="container">
+                    <input type="checkbox" id="checkbox-date">
+                    <label for="chackbox-date" class="text-style-1 checkbox-label">Date</label>
```
Comment (MaksimViarenich) [inline UI/index.html]: checkbox-date, not chackbox-date

File: UI/index.html [hunk 3]
```diff
@@ -0,0 +1,186 @@
...
+                </div>
+
+                <input type="text" id="ffrom" name="ffrom" placeholder="From">
```
Comment (MaksimViarenich) [inline UI/index.html]: Bad names for id and name

File: UI/index.html [hunk 4]
```diff
@@ -0,0 +1,186 @@
...
+                <input type="text" id="fto" name="fto" placeholder="To">
```
Comment (MaksimViarenich) [inline UI/index.html]: Bad names

Comment (MaksimViarenich) [inline UI/index.html]: For dates use type date, not text

**Comment by MaksimViarenich** [comment]
Merge, please, PR)

**Comment by denisbaranchik** [comment]
I thought you should approve first, but if you say so...

**Pull request merged**